### PR TITLE
Add default UKNOWN values to enums

### PR DESCRIPTION
**Pull request by gdbelvin**

### CODE REVIEW

File: core/api/type/type_proto/keymaster.proto [hunk 1]
```diff
@@ -48,8 +49,9 @@ message SigningKey {
 message VerifyingKey {
   // KeyStatus defines a key status.
   enum KeyStatus {
-    ACTIVE = 0;      // indicates a key that is currently being used.
-    DEPRECATED = 1;  // indicates a key that is not in use.
+    UKNOWN = 0;
+    ACTIVE = 2;      // indicates a key that is currently being used.
```
Comment (phad) [inline core/api/type/type_proto/keymaster.proto]: Should it be ACTIVE = 1, DEPRECATED = 3?
Or are you trying to keep numbers in sync with those in KeyStatus?
(If so, I wouldn't try)

Comment (gdbelvin) [reply core/api/type/type_proto/keymaster.proto]: I'm not explicitly trying to keep them in sync, but it's not a bad idea..

File: core/api/type/type_proto/keymaster.proto [hunk 2]
```diff
@@ -32,9 +32,10 @@ message Metadata {
 message SigningKey {
   // KeyStatus defines a key status.
   enum KeyStatus {
-    ACTIVE = 0;      // indicates a key that is currently being used.
-    INACTIVE = 1;    // indicates a key status that is not active.
-    DEPRECATED = 2;  // indicates a key that is not in use.
+    UKNOWN = 0;
```
Comment (phad) [inline core/api/type/type_proto/keymaster.proto]: typo: UNKNOWN

File: core/api/type/type_proto/keymaster.proto [hunk 3]
```diff
@@ -48,8 +49,9 @@ message SigningKey {
 message VerifyingKey {
   // KeyStatus defines a key status.
   enum KeyStatus {
-    ACTIVE = 0;      // indicates a key that is currently being used.
-    DEPRECATED = 1;  // indicates a key that is not in use.
+    UKNOWN = 0;
```
Comment (phad) [inline core/api/type/type_proto/keymaster.proto]: typo: UNKNOWN

File: core/api/type/type_proto/keymaster.proto [hunk 4]
```diff
@@ -48,8 +49,9 @@ message SigningKey {
...
+    ACTIVE = 1;      // indicates a key that is currently being used.
+    DEPRECATED = 3;  // indicates a key that is not in use.
```
Comment (phad) [inline core/api/type/type_proto/keymaster.proto]: I'm not sure of the benefit of aligning the enum values like this.  I'd just use 2 here.  There should not be any casting between the two KeyStatus types in any case!

File: core/api/type/type_proto/keymaster.proto (hunk 2 above)
Comment (gdbelvin) [reply core/api/type/type_proto/keymaster.proto]: Woops.  Thanks.

File: core/api/type/type_proto/keymaster.proto (hunk 4 above)
Comment (gdbelvin) [reply core/api/type/type_proto/keymaster.proto]: Yeah, no casting indeed.

**Pull request merged**